### DISCUSSION TITLE
Missing instructions on how to compile GRL files to GRB

### DISCUSSION THREAD

**Discussion by Homeblest** [hyperjumptech/grule-rule-engine#288] [Q&A] [answered]
I've searched high and low, looks like the instructions were removed at some point. 

Can somebody outline how to turn a GRL file into GRB please?

Cheers!

**Comment by newm4n** [comment, marked as answer]
https://github.com/hyperjumptech/grule-rule-engine/blob/master/docs/en/Binary_Rule_File_en.md
